credit-card/style.css

```
@@ -0,0 +1,88 @@
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  height: 100vh;
+  padding: 0;
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 16px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+p {
+  margin: 0;
+}
+
+.credit-card {
+  display: flex;
+  flex-direction: column;
+  background-color: #275ecc;
+  border-radius: 10px;
+  height: 180px;
+  width: 320px;
+  padding: 10px 20px;
+}
+
+.credit-card-front {
+  justify-content: flex-end;
+  margin-bottom: 15px;
+}
+
+.credit-card-front .information-wrapper {
+  margin-block: 10px;
+}
+
+.number {
+  font-size: 26px;
+}
+
+.information-wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.information-wrapper p:first-of-type {
+  text-transform: uppercase;
+  max-width: 50px;
+  font-size: 12px;
+}
+
+.credit-card-back .name {
+  font-family: "Qwitcher Grypen", cursive;
+  font-weight: 700;
+}
+
+.magnetic-strip{
```

```suggestion
.magnetic-strip {
```
Please be consistent in code formatting